tmc-prototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -352,110 +274,112 @@ def delete_device(self):
             try:
-                self._leaf_device_proxy[name].command_inout("SetStandbyLPMode")
-            except Exception as e:
-                print "Unexpected error in setting Standby mode on ", self._dish_leaf_node_devices[name]
-                self._read_activity_message = "Unexpected error in setting Standby mode on " + str(self._dish_leaf_node_devices[name])
-                print "Error message is: \n", e
-                self._read_activity_message = "Error message is: \n" + str(e)
-                self.devlogmsg("Unexpected error in executing StandByTelescope command!", 2)
-
-        # PROTECTED REGION END #    //  CentralNode.StandByTelescope
+                self._leaf_device_proxy[name].command_inout(CONST.CMD_SET_STANDBY_MODE)
+            except Exception as except_occured:
+                print CONST.ERR_EXE_STANDBY_CMD, self._dish_leaf_node_devices[name]
+                self._read_activity_message = CONST.ERR_EXE_STANDBY_CMD \
+                                              + str(self._dish_leaf_node_devices[name])
+                print CONST.STR_ERR_MSG, except_occured
+                self._read_activity_message = CONST.STR_ERR_MSG + str(except_occured)
+                self.devlogmsg(CONST.ERR_EXE_STANDBY_CMD, 2)
+        # PROTECTED REGION END #    //  CentralNode.standby_telescope
 
     @command(
     )
     @DebugIt()
     def StartUpTelescope(self):
         # PROTECTED REGION ID(CentralNode.StartUpTelescope) ENABLED START #
-        self.devlogmsg("StartUpTelescope command invoked from Central node", 4)
-        self._read_activity_message = "StartUpTelescope command invoked from Central node"
+        """ This command is to bring the Telescope into ON state from the STANDBY state."""
+        self.devlogmsg(CONST.STR_STARTUP_CMD_ISSUED, 4)
+        self._read_activity_message = CONST.STR_STARTUP_CMD_ISSUED
 
-        for name in range (0,len(self._dish_leaf_node_devices)):
+        for name in range(0, len(self._dish_leaf_node_devices)):
             try:
                 print self._leaf_device_proxy
-                self._leaf_device_proxy[name].command_inout("SetOperateMode")
-            except Exception as e:
-                print "Unexpected error in StartUp of ", self._dish_leaf_node_devices[name]
-                self._read_activity_message = "Unexpected error in StartUp of " + str(self._dish_leaf_node_devices[name])
-                print "Error message is: \n", e
-                self._read_activity_message = "Error message is: \n" + str(e)
-                self.devlogmsg("Unexpected error in executing StartUpTelescope command!", 2)
-        # PROTECTED REGION END #    //  CentralNode.StartUpTelescope
+                self._leaf_device_proxy[name].command_inout(CONST.CMD_SET_OPERATE_MODE)
+            except Exception as except_occured:
+                print CONST.ERR_EXE_STARTUP_CMD, self._dish_leaf_node_devices[name]
+                self._read_activity_message = CONST.ERR_EXE_STARTUP_CMD \
```

Comment:
Same as above.

Reply:
Ensure line length of 110 characters and split the line with parenthesis if exceeding. Commit id is 172675f9b8198800667bd184e60043698176d4b3.